tests/test_wait_for_it_to_not_raise_an_exception.py

```diff
@@ -0,0 +1,44 @@
+import time
+import unittest
+
+import wait_for_it_to
+
+try:
+    a = TimeoutError()
+except NameError:
+    TimeoutError = wait_for_it_to.TimeoutError
+
+
+class TestWaitForItToNotRaiseAnException(unittest.TestCase):
+    def test_wait_for_it_not_to_raise_an_exception_returns_immediatly_when_no_exception_raised(self):
+        def func():
+            pass
+
+        start = time.time()
+        wait_for_it_to.not_raise_an_exception(func)
+        end = time.time()
+        self.assertLessEqual(end - start, 0.01)
+
+    def test_raises_a_timeout_error_when_func_keeps_raising_exceptions(self):
+        def func():
+            raise Exception("this keeps on failing")
+
+        self.assertRaises(TimeoutError, wait_for_it_to.not_raise_an_exception, func, 1)
+
+    def test_raises_a_timeout_error_when_func_keeps_raising_exceptions_which_are_equal_to_the_sentinel_exception(self):
+        def func():
+            raise RuntimeError("this keeps on failing")
+
+        self.assertRaises(TimeoutError, wait_for_it_to.not_raise_an_exception, func, 1, RuntimeError)
+
+    def test_raises_an_exception_which_is_not_equal_to_the_sentinel_exception(self):
+        def func():
+            raise RuntimeError("this keeps on failing")
+
+        self.assertRaises(RuntimeError, wait_for_it_to.not_raise_an_exception, func, 1, EnvironmentError)
+
+    def test_takes_args_and_kwargs(self):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/studioj/wait_for_it/pullRequest?prid=4638435)